mongo/metadata.csv

```diff
@@ -225,14 +225,14 @@ mongodb.usage.update.time,gauge,,microsecond,,Total time spent performing update
 mongodb.usage.writeLock.countps,rate,,lock,,Number of write locks per second,0,mongodb,
 mongodb.usage.writeLock.count,gauge,,lock,,Number of write locks since server start (deprecated),0,mongodb,
 mongodb.usage.writeLock.time,gauge,,microsecond,,Total time spent performing write locks in microseconds,-1,mongodb,
-mongodb.tcmalloc.generic.current_allocated_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.generic.heap_size,gauge,,unit,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.aggressive_memory_decommit,gauge,,unit,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.central_cache_free_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.current_total_thread_cache_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.max_total_thread_cache_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.pageheap_free_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.pageheap_unmapped_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.thread_cache_free_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.transfer_cache_free_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.spinlock_total_delay_ns,gauge,,nanosecond,,,0,mongodb,
+mongodb.tcmalloc.generic.current_allocated_bytes,gauge,,byte,,Number of bytes used by the application.,0,mongodb,
+mongodb.tcmalloc.generic.heap_size,gauge,,byte,,Bytes of system memory reserved by TCMalloc.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.aggressive_memory_decommit,gauge,,unit,,Status of aggressive memory decommit mode.,0,mongodb,
```

Comment:
It seems that `aggressive_memory_decommit` is a boolean, not `unit`, we can leave the unit column empty since there is not adequate unit for booleans.

Reply:
`unit` would be a default unit, it's general. But I can remove it

Reply:
But IMO it's not suitable for a boolean since `unit` seems to be for things that can be counted.